internal/cli/atlas/performanceadvisor/namespaces/list.go

```diff
@@ -54,18 +54,20 @@ func (opts *ListOpts) Run() error {
 	if err != nil {
 		return err
 	}
-	r, err := opts.store.PerformanceAdvisorNamespaces(opts.ConfigProjectID(), host, opts.newNamespaceOptions())
+	r, err := opts.store.PerformanceAdvisorNamespaces(opts.newNamespaceOptions(opts.ConfigProjectID(), host))
 	if err != nil {
 		return err
 	}
 
 	return opts.Print(r)
 }
 
-func (opts *ListOpts) newNamespaceOptions() *atlas.NamespaceOptions {
-	return &atlas.NamespaceOptions{
-		Since:    opts.since,
-		Duration: opts.duration,
+func (opts *ListOpts) newNamespaceOptions(project, host string) *atlas.ListSlowQueryNamespacesApiParams {
+	return &atlas.ListSlowQueryNamespacesApiParams{
+		GroupId:   project,
+		ProcessId: host,
+		Since:     &opts.since,
+		Duration:  &opts.duration,
```

Comment:
😮

Reply:
Not sure what you mean

Reply:
I assume based on the new params, this is the new change to the v2 SDK to pass the params as a struct. 
Nice improvement!